Models/Entry.cs

```diff
@@ -1,14 +1,15 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
+﻿using Newtonsoft.Json;
+using System;
 
 namespace WhisperLeaderboard.Models
 {
-    public class Entry
+    public class Entry : IEquatable<Entry>
```

Comment:
de la doc :
If you implement IEquatable<T>, you should also override the base class implementations of Equals(Object) and GetHashCode()